### DISCUSSION TITLE
User journey for a researcher

### DISCUSSION THREAD

**Discussion by flefevre** [open-webui/open-webui#3278] [Ideas]
It will be very useful if open WebUI has some specific action button to help researchers in their daily life

A button to record a live meeting in order to make the stt note directly 
A button to summarise such meeting and highlighting the decision made
A button to transform a chat message selection into a summary in order to store it in a memory database 
A button to summarise a scientific publication with a service to retrieve additional data on the laboratory web site

Etc.

Perhaps such actions buttons come from 'open webui pipeline'

**Comment by TomLucidor** [comment]
Don't just wing it, help out here as well https://github.com/open-webui/open-webui/discussions/9321
